packages/create-cloudflare/e2e-tests/workers.test.ts

```diff
@@ -195,11 +195,11 @@ describe
 							expect(wranglerPath).toExist();
 
 							const tomlPath = join(project.path, "wrangler.toml");
-							const jsonPath = join(project.path, "wrangler.json");
+							const jsoncPath = join(project.path, "wrangler.jsonc");
 
 							try {
-								expect(jsonPath).toExist();
-								const config = readJSON(jsonPath) as { main?: string };
+								expect(jsoncPath).toExist();
+								const config = readJSON(jsoncPath) as { main?: string };
```

Comment:
super nit: don't we have the type for the config?

Reply:
Not in C3, I don't think? It's defined in Wrangler, and right now there's no dependency between the two packages